use-cases.md

```diff
@@ -119,9 +126,28 @@ After a customer has checked out from a reservation, reputation management syste
 
 Upon a Reputation management system associating feedback with a customer the [Update customer](operations/customers.md#update-customer) operation should be added to the customer profile in Mews. The customer classification type `Previous complaint` is one which should be used when negative feedback has been received. Further to this, keywords from the complaint and a url to the survey or Tripadvisor post can be added to the customer `notes`.
 
-## Mobile Key Systems
+### Marketing Opt-out
+
+A customer has the ability to opt-out of marketing communciation. As Reputation management systems are also commonly used to launch marketing campaigns, the integration should ensure that if the option, `SendMarketingEmails` in the customer object is non existant then the customer is unsubscribed to any marketing.
```

Comment:
A customer has the ability to opt-out of marketing communication. As reputation management systems are also commonly used to launch marketing campaigns, the integration should ensure that if the option, `SendMarketingEmails`, in the customer object is non existent then the customer is unsubscribed from any marketing.

Reply:
"Reputation" is an adjective and shouldn't be capitalized
"existant" is spelled "existent" and "non-existent" is hyphenated.  I don't know if existant is specific to programming.